gradle.properties

```diff
@@ -1,5 +1,7 @@
 group=org.partiql
-version=0.6.3-SNAPSHOT
+version=0.6.3
 # Required for the maven publish plugin--these properties must be present even if they contain placeholder values.
 ossrhUsername=EMPTY
 ossrhPassword=EMPTY
+
+org.gradle.jvmargs=-Xmx4096M -XX:MaxMetaspaceSize=2G
```

Comment:
Following suggestion from https://github.com/Kotlin/dokka/issues/1405 about OOM errors seen in some `dokkaHtml` steps.